validation/src/constants.rs

```diff
@@ -84,4 +84,6 @@ pub mod test {
     pub const TESTNET_HEADER_2132555: &str = "004000200e1ff99438666c67c649def743fb82117537c2017bcc6ad617000000000000007fa40cf82bf224909e3174281a57af2eb3a4a2a961d33f50ec0772c1221c9e61ddfdc061ffff001a64526636";
     /// Testnet 00000000383cd7fff4692410ccd9bd6201790043bb41b93bacb21e9b85620767
     pub const TESTNET_HEADER_2132556: &str = "00000020974f55e77dff100bc252a01aa7b00d16736c6e04a091b03be200000000000000c44f2d69fc200c4a2211885000b6b67512f42c1bec550f3754e103b6c4046e05a202c161ffff001d09ec1bc4";
+    /// Mock time used as the current time.
+    pub const MOCK_CURRENT_TIME: u64 = 2_634_590_600;
```

Comment:
Why is this constant here? It's not used in this file, and it's also not used in production. I'd keep this constant with the tests where it's used.

Reply:
Those are constants for `mod test`,  and all constants from above are used only in tests in `header.rs`.